dff/pipeline/pipeline/pipeline.py

```diff
@@ -13,16 +13,17 @@
 class, which is defined in the Component module. Together, these classes provide a powerful and flexible way
 to structure and manage the messages processing flow.
 """
+from __future__ import annotations
 import asyncio
 import logging
-from typing import Union, List, Dict, Optional, Hashable, Callable
+from typing import Union, List, Dict, Optional, Hashable, Callable, TYPE_CHECKING
 
 from dff.context_storages import DBContextStorage
 from dff.script import Script, Context, ActorStage
 from dff.script import NodeLabel2Type, Message
 from dff.utils.turn_caching import cache_clear
 
-from dff.messengers.common import MessengerInterface, CLIMessengerInterface
+import dff.messengers.common
```

Comment:
Why can't we import the class itself (as we always do)?

Reply:
For some reason I thought that there is a cyclic import between `pipeline.types` and `messengers.common.interface`. That is not the case.
Fixed in ce3a8a2f7936d0482cec7d14fa460ce3971e0ca3.